src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -89,6 +93,53 @@ public void GenerateOutput()
             Utils.AddProjectsToSolution(modelprojectDir, modelProjectName, consoleAppProjectDir, consoleAppProjectName, solutionPath);
         }
 
+        public void GenerateAzureRemoteImageOutput()
+        {
+            // Get Namespace
+            var namespaceValue = Utils.Normalize(_settings.OutputName);
+            var labelType = _columnInferenceResult.TextLoaderOptions.Columns.Where(t => t.Name == _settings.LabelName).First().DataKind;
+            Type labelTypeCsharp = Utils.GetCSharpType(labelType);
+
+            // Generate Model Project
+            var modelProjectContents = GenerateAzureAttachImageModelProjectContents(namespaceValue);
+
+            var modelProjectDir = Path.Combine(_settings.OutputBaseDir, $"{_settings.OutputName}.Model");
+            var modelProjectName = $"{_settings.OutputName}.Model.csproj";
+
+            // Get Normalize Mapping
+            var normalizeContent = new NormalizeMapping()
+            {
+                Namespace = namespaceValue,
+                Target = _settings.Target,
+            }.TransformText();
+            normalizeContent = Utils.FormatCode(normalizeContent);
+
+            // Write files to disk
+            Utils.WriteOutputToFiles(modelProjectContents.ModelInputCSFileContent, "ModelInput.cs", modelProjectDir);
+            Utils.WriteOutputToFiles(modelProjectContents.ModelOutputCSFileContent, "ModelOutput.cs", modelProjectDir);
+            Utils.WriteOutputToFiles(modelProjectContents.ConsumeModelCSFileContent, "ConsumeModel.cs", modelProjectDir);
+            Utils.WriteOutputToFiles(modelProjectContents.ModelProjectFileContent, modelProjectName, modelProjectDir);
+            Utils.WriteOutputToFiles(normalizeContent, "NormalizeMapping.cs", modelProjectDir);
+
+            // Generate ConsoleApp Project
+            var consoleAppProjectContents = GenerateConsoleAppProjectContents(namespaceValue, labelTypeCsharp,
+                false, false, false, true, false, false, true, true);
```

Comment:
Named parameters would make this more readable

Reply:
Yeah I agree, but It's just the old test case, so I'd better not change that